content/pages/constitution.md

```diff
@@ -172,6 +178,39 @@ All committee memebers are free to delegate their responsibilities to others as
 as they see fit, but are ultimately responsible for the completion of their
 assigned duties.
 
+## ii. Volunteers ##
+
+1.  All individuals are eligible to become volunteers.
```

Comment:
Yes, we've had male volunteers at many of our events that are open to the public. We also have had non-student volunteers as well.

I might want to rewrite this line though: "All individuals are eligible to become a volunteer."